### ISSUE TITLE
Require mobile web designs at DI and MPR, and user flow at MPR

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#89243]
## User Story
As a Governance team member supporting the Collaboration Cycle, I want VFS teams to prioritize mobile web designs at the DI and MPR so that I can review the their product and give them helpful feedback.

As a Governance team member supporting the Collaboration Cycle, I want VFS teams to share their user flows at Midpoint Review, so that I can verify changes in flow from the DI, or so that I can see their intended flow if they skipped DI.

Assignee: @briandeconinck 
Peer Reviewer: @erinrwhite 

## Description
Governance team still sees many VFS teams come to DI and/or MPR with desktop designs only. While this is helpful, and better than no design at all, desktop designs don't allow Governance to fully test and/or understand the product. When VFS teams design for mobile first (instead of web first) they're solving for the problem in limited space, and are solving for accessibility concerns that don't exist in desktop. Also, mobile web traffic has grown significantly. We know that many of our veterans are accessing VA.gov via their mobile device vs. their computer. 

Mobile first is important bc smartphones ~ 400% zoom, also designing for low vision users. Sometimes teams will take desktop features not available on mobile. Younger, blind veterans don't have desktop typically. Can do everything on mobile.

Because of all of the above, Governance will begin requiring VFS teams to share mobile web designs at DI and MPR. 

In addition, we will also begin to ask VFS teams to share an up-to-date user flow at Midpoint Review. We do currently require a user flow at Design Intent. Of the 38 Midpoint Review's that have occurred so far in 2024 (as of 8/29), 13 of them did not have a Design Intent. This means that Governance potentially did not see a user flow for 13 of those products, making it more difficult for us to understand the products purpose and intentions. It is also possible that VFS teams skip DI intentionally and go straight to MPR because of the laxed requirements. 

### Live pages
- [DI Touchpoint page](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent) (Platform Website)
- [MPR Touchpoint page](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review) (Platform Website)
- [Change history ](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/change-history) (Platform Website)
- [Collab Cycle kickoff- code blocks](https://vfs.atlassian.net/wiki/spaces/PPS/pages/3387457549/Collaboration+Cycle+Kickoff+-+Code+Blocks) (Confluence VFS facing)

## Drafts
- [Draft DI touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045273/89243+-+Design+Intent)
- [Draft MPR touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045293/89243+-+Midpoint+Review)
- [Draft code blocks](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045313/89243+-+Collaboration+Cycle+Kickoff+-+Code+Blocks)
- [Draft change announcements](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477406043/89243+-+Change+announcements)

## Tasks
- [x] Make copies/drafts to update platform website pages to accurately reflect required artifacts for design intent and midpoint review
- [x] Draft a change announcement on slack and platform website for announcing this change to VFS teams
- [x] Get peer review on change announcements on updated pages and code blocks
- [x] Publish platform website pages and updated code blocks
- [x] Make announcement

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] Governance team has added mobile web designs as a required artifact to the ~~Design Intent and~~ Midpoint Review
   - Note 9/25: See comment section for discussion. Governance team decided that we should not _require_ mobile designs at Design Intent for the reason that we don't _require_ any designs at Design Intent.  If a VFS team does decide to share designs at the Design Intent, we will specify that, "If you include designs, make sure they're mobile-first" 
- [x] Governance team has added user flows as a required artifact to the Midpoint Review
- [x] The code blocks for DI and MPR have been updated to reflect the new required artifacts 
- [x] The DI and MPR touchpoint pages have been updated to reflect the new required artifacts
- [x] We have announced to VFS teams the changes in required artifacts at DI and MPR

## Team Notification
- [ ] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [x] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [ ] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by erinrwhite** [comment]
Refined this ticket a little, making drafts of Platform website pages and code blocks

**Comment by briandeconinck** [comment]
@erinrwhite This is ready for peer review!
- [Draft DI touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045273/89243+-+Design+Intent)
- [Draft MPR touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045293/89243+-+Midpoint+Review)
- [Draft code blocks](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477045313/89243+-+Collaboration+Cycle+Kickoff+-+Code+Blocks)
- [Draft change announcements](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3477406043/89243+-+Change+announcements)

**Comment by erinrwhite** [comment]
These look really good. Be sure to grab the latest version of the [code blocks](https://vfs.atlassian.net/wiki/spaces/PPS/pages/3387457549/Collaboration+Cycle+Kickoff+-+Code+Blocks) page.

Overall I think we should be more broad in our artifacts guidance on platform/github and say "mobile first" but not specifically say 320px breakpoint, for a couple of reasons...

- It's a level of specificity that we don't have in our other artifact guidance for DI/MPR
- It's also more specific than what we say outright in VADS. Our current guidance on VADS is a bullet point about mobile-first design on the [Design Principles page](https://design.va.gov/about/principles) and a table on our [CSS breakpoints guidance](https://design.va.gov/foundation/breakpoints) page, but overall the mobile-first language could be strengthened. Ideally we should have something about this either in our [For Designers](https://design.va.gov/about/designers/) section, or in our Figma documentation (maybe we already do?).

**tl;dr:** I recommend leaving the 320px-specific guidance out for now, but am also fine with it shipping as-is! I also recommend that as a next step we work with the DST to codify the specifics of mobile-first practice in VADS or in the Platform website guidance for designers.

**Comment by shiragoodman** [comment]
I tend to agree with Erin here. Let's roll the language back to align with what's in VADS... at least for the scope of this ticket. I'm open to modifying the language across the board to be more specific with what we man by "mobile design" so @briandeconinck if that's something you think would be valuable, please open a new ticket and tag me in it.

I do have a question for both of you wrt the changes made to the Design Intent. This ticket specifies that we should _require_ mobile web designs at Design Intent, however right now at DI, we don't _require_ any designs - just a user flow. I'm assuming that's why you made the change where you did Brian, but I'm not entirely sure it meets the acceptance criteria. 

What do you 2 think? Is _requiring_ mobile web designs at DI the right move if we don't even require designs currently at DI? Tagging @humancompanion-usds for his opinion, however I'd love to hear your opinions.

**Comment by erinrwhite** [comment]
@shiragoodman thanks for this! I agree, we shouldn't be requiring any designs at Design Intent - but if folks have them to share, they should be mobile-first. I think @briandeconinck's changes thread that needle well: "If you include designs, make sure they're mobile-first"

**Comment by briandeconinck** [comment]
Yeah, as @erinrwhite says, this was my attempt to thread the needle. I really like it when teams come to DI with just user flows, and I don't think we should change our artifact requirements to require wireframes. I'd be more interested in changing the acceptance criteria on this ticket! 😄 

As for the 320px breakpoint --- I'm fine taking out that specific value. I was worried about that ambiguity of what "counts" as mobile, but on reflection I don't think the DI/MPR pages are the right place to do that.

**Comment by shiragoodman** [comment]
I'll made some changes to the AC. Thank you both!! I think this is on the right track.

**Comment by shiragoodman** [comment]
@erinrwhite @briandeconinck please be sure to coordinate with @it-harrison re: the change announcement as I'd like it to coincide with the work Ian is doing to update the open Collab Cycle request tickets themselves via ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/91807.

Ideally we should execute the script to update the open tickets and then post the change announcement.

**Comment by briandeconinck** [comment]
@erinrwhite Confluence updated with edits per the discussion above. If you can give a quick review, I can get those updated on Platform website and then pass the baton to @it-harrison for updating tickets!

**Comment by erinrwhite** [comment]
@briandeconinck looks good to go! Don't forget to update the "last updated" time on the Platform pages.

**Comment by briandeconinck** [comment]
Platform website pages for Design Intent and Midpoint Review have been updated, should be published later today.

[Code blocks](https://vfs.atlassian.net/wiki/spaces/PPS/pages/3387457549/Collaboration+Cycle+Kickoff+-+Code+Blocks) have been updated. These should be ready for @it-harrison in #91807.

Change announcements are on hold pending completion of #91807. Since I'll be out of office this afternoon that means this ticket will definitely carry over into the next sprint --- but like, just barely! 😄

**Comment by briandeconinck** [comment]
Announcement posted in #vfs-change-announcements.

@erinrwhite I think this is ready to close out. Would you give it a quick final review to verify/check off some boxes?

**Comment by erinrwhite** [comment]
Looking good @briandeconinck , all AC and peer review checkboxes have been checked. Good to close.
